skcosmo/model_selection/_split.py

```diff
@@ -68,7 +68,7 @@ def train_test_split(*arrays, **options):
         else:
             options["test_size"] = None
             train_sets = sklearn.model_selection.train_test_split(*arrays, **options)[
-                1::2
+                ::2
```

Comment:
Is this just a superficial change? i.e. just changing the test to the even samples and the train to the odd? If so, is there any reason for this change?

Reply:
I was choosing the train test when actually the train test should have been chosen and the other way around for the test set. The output order is (TRAIN_1, TEST_1, TRAIN_2, TEST_2, ...).
When I added your suggestion 
```train_size == test_size == 0.8*(total_size)```
I realized it, because the outputs were 0.2*(total_size)

Reply:
added #43